src/test/java/com/rapid7/container/analyzer/docker/service/DockerImageAnalyzerServiceTest.java

```diff
@@ -7,15 +7,18 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import org.junit.jupiter.api.Test;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class DockerImageAnalyzerServiceTest {
 
   @Test
   public void test() throws IOException {
+    Logger.getGlobal().setLevel(Level.ALL);
```

Comment:
we can set this via a log config on classpath in test too

Reply:
This is a left-over from some manual testing, I've removed it now